docs/_includes/skeleton-loader.html

```diff
@@ -0,0 +1,384 @@
+          </li>
+          <li>
+            <div class="skeleton--tile">
+              <div class="skeleton-image-container">
+                <div class="skeleton skeleton--image"></div>
+              </div>
+              <div class="skeleton skeleton--text-small skeleton--text-multiline"></div>
+            </div>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </div>
+  {% highlight html %}
+  <div style="width: 900px; background-color: lightyellow; border: 1px dashed orange;">
+    <ul class="skeleton--tiles-layout" role="img" aria-label="loading" data-columns="3">
+      <li>
+        <div class="skeleton--tile">
+          <div class="skeleton-image-container">
+            <div class="skeleton skeleton--image"></div>
+          </div>
+          <div class="skeleton skeleton--text-small skeleton--text-multiline"></div>
+        </div>
+      </li>
+      <li>
+        <div class="skeleton--tile">
+          <div class="skeleton-image-container">
+            <div class="skeleton skeleton--image"></div>
+          </div>
+          <div class="skeleton skeleton--text-small skeleton--text-multiline"></div>
+        </div>
+      </li>
+      <li>
+        <div class="skeleton--tile">
+          <div class="skeleton-image-container">
+            <div class="skeleton skeleton--image"></div>
+          </div>
+          <div class="skeleton skeleton--text-small skeleton--text-multiline"></div>
+        </div>
+      </li>
+      <li>
+        <div class="skeleton--tile">
+          <div class="skeleton-image-container">
+            <div class="skeleton skeleton--image"></div>
+          </div>
+          <div class="skeleton skeleton--text-small skeleton--text-multiline"></div>
+        </div>
+      </li>
+      <li>
+        <div class="skeleton--tile">
+          <div class="skeleton-image-container">
+            <div class="skeleton skeleton--image"></div>
+          </div>
+          <div class="skeleton skeleton--text-small skeleton--text-multiline"></div>
+        </div>
+      </li>
+      <li>
+        <div class="skeleton--tile">
+          <div class="skeleton-image-container">
+            <div class="skeleton skeleton--image"></div>
+          </div>
+          <div class="skeleton skeleton--text-small skeleton--text-multiline"></div>
+        </div>
+      </li>
+    </ul>
+  </div>
+  {% endhighlight %}
+  <h5>Variant: AI</h5>
+  <div class="demo">
+    <div class="demo__inner">
+      <div style="width: 225px;">
+        <div class="skeleton--tile" role="img" aria-label="loading">
+          <div class="skeleton-image-container">
+            <div class="skeleton skeleton-ai-gradient-blue-subtle skeleton--image"></div>
+          </div>
+          <div class="skeleton skeleton-ai-gradient-blue-subtle skeleton--text-small"></div>
+        </div>
+      </div>
+    </div>
+  </div>
+  {% highlight html %}
+  <div style="width: 225px;">
+    <div class="skeleton--tile" role="img" aria-label="loading">
```

Comment:
This is another BEM violation. Can't have `skeleton--tile` modifier without a base class.